### DISCUSSION TITLE
Uno-check .Net Workload install failed, Error due to package signing

### DISCUSSION THREAD

**Discussion by znbrain** [unoplatform/uno#14022] [Q&A]
I use uno-check 1.16.0.0
Everything works except for installing the workloads

![image](https://github.com/unoplatform/uno/assets/5909446/04d6fd96-757f-401f-9487-9636cc84627c)

![image](https://github.com/unoplatform/uno/assets/5909446/39e0156e-d175-45c6-983d-350733c0b136)
...
![image](https://github.com/unoplatform/uno/assets/5909446/1c2f8491-3f24-4f41-aa45-d150ad245576)

I have checked the workloads in Visual Studio Installer whit the "Getting startet Documentation", but i cannot see what i have doing wrong.

I use Visual Studio 17.7.5 on Windows 10

**Comment by jeromelaban** [comment]
Thanks for the report. We're keeping track of this issue here: https://github.com/unoplatform/uno.check/issues/182

**Comment by limeniye** [comment]
It looks so similar as the following answer: https://github.com/unoplatform/uno/discussions/13891#discussioncomment-7184971
I recommend you to try this command:
```
dotnet workload install maui
```
